yajsapi/market/decorations_builder.ts

```diff
@@ -0,0 +1,69 @@
+import { MarketProperty } from "ya-ts-client/dist/ya-payment/src/models/market-property";
+import { MarketDecoration } from "ya-ts-client/dist/ya-payment/src/models/market-decoration";
+
+export enum ComparisonMethod {
```

Comment:
Maybe `ComparisionOperator` would be a better name..? like in https://www.mongodb.com/docs/manual/reference/operator/query-comparison/